### PR TITLE
MISC-415 private locations

### PR DESCRIPTION
**Pull request by Isammoc**
Renaming

First PR with automated screenshots

### CODE REVIEW

**Comment by ccedric** [comment]
- api token / users modal: team roles should be selected
- all tables: each line is cropped on the right
- team modal: why is the "save" button not enabled?
- dedicated ips/simulations: table is empty
- We should rename the user "cypress-dev", it is visible in 'admin-users.png'
- credit-comsumption: missing chart
- on some modals: last input is still selected
- linked-accounts: we can see okta-test-ggaly
- samples: missing mvn img

**Review by slandelle — CHANGES_REQUESTED** [review]
The automated generation looks awesome. However, documentation.png shows an error where Documentation overlaps with the link icon. I don't have it on my computer.

**Review by slandelle — CHANGES_REQUESTED** [review]
~~Also, lots of titles sound like Hodor speaking:  "Upload Package", "Create Simulation". Shouldn't be "Upload a Package" and "Create a Simulation", etc?~~

**Review by franciscotco — COMMENTED** [review]
Missing space at the right of almost all images.
There are no dedicated IPs to show in the current organization.
NavBar dropdown, organization menu, and profile menu are not rendered.

File: content/reference/plugins/grafana/index.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
     - `usr+`: Users arrival count per second.
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:111]: `admin/api_tokens/create-token.png`:
team roles should be selected

File: content/reference/plugins/grafana/index.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
...
     - `usr-`: Users termination count per second.
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:112]: `admin/users/users-invite.png`: team roles should be selected

File: content/reference/plugins/grafana/index.md [hunk 3]
```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
...
     - `usrActive`: Concurrent users count per second.
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:113]: `admin/teams/create-team.png`: why is the "save" button not enabled?

File: content/reference/plugins/grafana/index.md [hunk 4]
```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
...
-- **Injectors metrics:**
+- **Load Generators metrics:**
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:114]: all tables: each line is cropped on the right

File: content/reference/plugins/grafana/index.md [hunk 5]
```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
...
     - `gcCount`: GC count per second.
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:115]: `user/dedicated_ips/dedicated-ips.png`: table is empty

File: content/reference/plugins/grafana/index.md [hunk 6]
```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
...
     - `gcTime`: GC time per second.
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:116]: `user/organization/admin-users.png`: We should rename the user "cypress-dev",

File: content/reference/plugins/grafana/index.md [hunk 7]
```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
...
     - `heapUsed`: Heap used in Mb.
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:117]: `user/organization/credits-consumption.png`: missing chart

File: content/reference/plugins/grafana/index.md [hunk 8]
```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:175]: on some modals: last input is still selected

File: content/reference/plugins/grafana/index.md [hunk 9]
```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
 
 ### Part 2
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:176]: linked-accounts: we can see okta-test-ggaly

File: content/reference/plugins/grafana/index.md [hunk 10]
```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
...
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:177]: samples: missing mvn img

File: content/reference/plugins/grafana/index.md [hunk 11]
```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
...
 
-| Metrics name                   | remote             | hostname           | injector           | scope              |
+| Metrics name                   | remote             | hostname           | load generator     | scope              |
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:178]: documentation.png shows an error where Documentation overlaps with the link icon. I don't have it on my computer.

File: content/reference/plugins/grafana/index.md [hunk 12]
```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
...
 ---------------------------------|--------------------|--------------------|--------------------|--------------------|
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:179]: Missing space at the right of almost all images.

File: content/reference/plugins/grafana/index.md [hunk 13]
```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
...
 | `req.<Percentiles>`            |                    |                    |                    | {{< icon check >}} |
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:180]: NavBar dropdown not rendered

File: content/reference/plugins/grafana/index.md [hunk 14]
```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
...
 | `requests`                     |                    |                    |                    |                    |
```
Comment (Isammoc) [inline content/reference/plugins/grafana/index.md:181]: organization menu not rendered

File: content/reference/user/api/index.md [hunk 15]
```diff
@@ -21,6 +21,6 @@ IDs can be retrieved on table either under kebab menu, or on the right of the ta
```
Comment (Isammoc) [inline content/reference/user/api/index.md:21]: profile menu not rendered

File: content/reference/plugins/grafana/index.md (hunk 1 above)
Comment (Isammoc) [reply content/reference/plugins/grafana/index.md:111]: Impossible for now.
Waiting for gatling/frontline-cloud#2174

File: content/reference/plugins/grafana/index.md (hunk 2 above)
Comment (Isammoc) [reply content/reference/plugins/grafana/index.md:112]: Impossible for now.
Waiting for gatling/frontline-cloud#2174

File: content/reference/plugins/grafana/index.md (hunk 4 above)
Comment (Isammoc) [reply content/reference/plugins/grafana/index.md:114]: done (thanks to prior work from @notdryft )

File: content/reference/plugins/grafana/index.md (hunk 5 above)
Comment (Isammoc) [reply content/reference/plugins/grafana/index.md:115]: @ccedric What do you want? Asking for dedicated_ips for this account or using the demo account?

File: content/reference/plugins/grafana/index.md (hunk 9 above)
Comment (Isammoc) [reply content/reference/plugins/grafana/index.md:176]: What do you suggest?
* rename it in the dev instance?
* rename it in the html?
* remove this item?

Comment (Isammoc) [reply content/reference/plugins/grafana/index.md:176]: @ccedric

File: content/reference/user/dedicated_ips/index.md [hunk 16]
```diff
@@ -1,38 +1,38 @@
...
 
-You can enable the use of dedicated IPs when [configuring simulation pools]({{< ref "../simulations#step-3-pools-configuration" >}}).
+You can enable the use of dedicated IP addresses when [configuring simulation locations]({{< ref "../simulations#step-3-locations-configuration" >}}).
 
-When starting a run of a simulation configured to use dedicated IPs,
-if you have enough dedicated IPs available to satisfy the size of the configured pools,
-they will be reserved for the run duration.  If you don’t have enough dedicated IPs available, the run won't start.
+When starting a run of a simulation configured to use dedicated IP addresses,
+if you have enough dedicated IP addresses available to satisfy the size of the configured locations,
+they will be reserved for the run duration.  If you don’t have enough dedicated IP addresses available, the run won't start.
```
Comment (slandelle) [inline content/reference/user/dedicated_ips/index.md:38]: documentation.png still has a margin issue.

Comment (slandelle) [reply content/reference/user/dedicated_ips/index.md:38]: credits-consumption.png is still missing the legends

Comment (slandelle) [reply content/reference/user/dedicated_ips/index.md:38]: overview.png is supposed to display the original landing page with the warning message when there's no package nor simulation.

**Review by slandelle — CHANGES_REQUESTED** [review]

File: content/reference/user/dedicated_ips/index.md (hunk 16 above)
Comment (Isammoc) [reply content/reference/user/dedicated_ips/index.md:38]: * documentation.png => Thank @franciscotco!
* credits-consumption.png => bug fixed but not deployed (even on dev environment)
* overview.png => created another organization to have empty data

File: content/reference/plugins/grafana/index.md (hunk 9 above)
Comment (ccedric) [reply content/reference/plugins/grafana/index.md:176]: I think we can rename it to "My Enterprise Private Login"

File: content/reference/user/api/index.md [hunk 17]
```diff
@@ -21,6 +21,6 @@ IDs can be retrieved on table either under kebab menu, or on the right of the ta
 
 Some information before using the public API:
```
Comment (ccedric) [inline content/reference/user/api/index.md:22]: kebab-copy.png: the image is cropped, we can't see all the options

File: content/reference/user/package_gen/index.md [hunk 18]
```diff
@@ -122,17 +122,17 @@ Assuming the CSV file contains 1000 entries, and you run your simulation on 3 Ga
...
 
-In your simulation code, you can resolve the name of the pool in which the injector running the code is deployed:
+In your simulation code, you can resolve the name of the location in which the load generator running the code is deployed:
 
 ```scala
-val poolName = System.getProperty("gatling.frontline.poolName")
-val baseUrl = if (poolName == "Europe - Paris") "https://example.fr" else "https://example.com"
+val locationName = System.getProperty("gatling.frontline.poolName") // pool is the former name of location
+val baseUrl = if (locationName == "Europe - Paris") "https://example.fr" else "https://example.com"
 ```
```
Comment (ccedric) [inline content/reference/user/package_gen/index.md:136]: profile-menu.png: nit: we can clearly see the profile menu, but the pie chart we see below it is not representative to what the user will see

**Review by tpetillot — APPROVED** [review]
🎊

File: content/reference/user/organization/index.md [hunk 19]
```diff
@@ -35,7 +35,7 @@ Click on the pen icon to edit the **Organization name**.
 {{< img src="credits-consumption.png" alt="Organization credit informations" >}}
```
Comment (franciscotco) [inline content/reference/user/organization/index.md:35]: There is a problem with the pie chart on the `credits-consumption.png` image:

[images/credits-consumption.png](https://github.com/gatling/frontline-cloud-doc/pull/115/files#diff-349af8b4f471bd9a0eb171f426f16be744ef3bdcf33d386536f8319a1531ee9d)

Comment (Isammoc) [reply content/reference/user/organization/index.md:35]: thx

**Review by franciscotco — APPROVED** [review]

File: content/reference/user/organization/index.md [hunk 20]
```diff
@@ -88,7 +88,7 @@ Plans view history.
 This section is only available to Organization System Admins.
 {{< /alert >}}
 
-This page shows all available offers for your organization. You can choose the number of credits for your offer. A credit represents a minute of usage of one Gatling Load injector.
+This page shows all available offers for your organization. You can choose the number of credits for your offer. A credit represents a minute of usage of one Gatling Load generator.
```
Comment (notdryft) [inline content/reference/user/organization/index.md:91]:
```suggestion
This page shows all available offers for your organization. You can choose the number of credits for your offer. A credit represents a minute of usage of one Gatling load generator.
```

**Review by notdryft — CHANGES_REQUESTED** [review]

**Review by notdryft — APPROVED** [review]

**Pull request merged**